src/main/java/conflux/dex/service/blockchain/TransactionConfirmationMonitor.java

```diff
@@ -114,7 +114,7 @@ public void confirmTransactions() {
 		try {
 			this.confirmTransactionsUnsafe();
 		} catch (Exception e) {
-			logger.debug("failed to confirm transaction", e);
```

Comment:
Exceptions(NullPointerException for this time, see below) are erased since logging level is usually above `debug`.